# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/kde/kdelibs/qt-3.3-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


A official kde-3.2 + qt-3.3 adaption patch sent by:

  - Raphael Langerhorst <raphael-langerhorst@gmx.at> 

--- kdelibs-3.1.95/kdefx/kstyle.cpp.orig	2004-01-17 13:53:01.000000000 +0100
+++ kdelibs-3.1.95/kdefx/kstyle.cpp	2004-01-20 08:37:44.625573288 +0100
@@ -971,14 +971,6 @@
 		case PM_MaximumDragDistance:
 			return -1;
 
-#if QT_VERSION >= 0x030300
-		case PM_MenuBarItemSpacing:
-			return 3;
-
-		case PM_ToolBarItemSpacing:
-			return 0;
-#endif
-
 		default:
 			return QCommonStyle::pixelMetric( m, widget );
 	}
